notebooks/0.data_splits/data_splits.ipynb

```diff
@@ -0,0 +1,2808 @@
+{
```

Comment:
<p>Consider adding a stratify statment here to stratify the splits across injury type. [sklearn - train_test_split](<a href="https://scikit-learn.org/stable/modules/generated/sklearn.model_selection.train_test_split.html" target="_blank">https://scikit-learn.org/stable/modules/generated/sklearn.model_selection.train_test_split.html</a>)</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/WayScience/Cytotoxic-Nuisance-Metadata-Analysis/pull/2/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='41'/>

Reply:
I didn't know about this! Thanks! aadded